chance/p5js/colorfulcircles/index.md

```diff
@@ -0,0 +1,11 @@
+---  
+layout: p5js
+author: Malte Völkner
+author-url: https://github.com/MalteVoelkner
+title: Colorfulcircles
+tags: [P5.js, chance, Law of Chance]
```

Comment:
How am I supposed to do that? In my .md file?
I can't find anything about a definition in your guide.